enhancements/insights/pulling-data-from-ocm.md

```diff
@@ -0,0 +1,139 @@
+---
+title: pulling-and-exposing-data-from-ocm
+authors:
+  - "@tremes"
+reviewers:
+  - "@sbose78"
+  - "@inecas"
+  - "@petli-openshift"
+  - "@smarterclayton"
+approvers:
+  - "@sbose78"
+  - "@smarterclayton"
+creation-date: 2021-03-04
+last-updated: 2021-03-09
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Insights Operator pulling and exposing data from the OCM API
```

Comment:
This is a really generic title. Are we only going to expose the certs, or are there plans to expose other data?

Reply:
OK let me change it to mention just the certs. Thanks

Reply:
Sounds good. The filename should be updated, too.

Reply:
Done.